docs/tutorial/gpu_user_guide.md

```diff
@@ -0,0 +1,104 @@
+### GPU User Guide
+
+#### Step 1: Prepare the system for NVIDIA GPU support
+
+To enable NVIDIA GPU support, follow these steps to configure the system:
+
+```bash
+distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
```

Comment:
Where do we run the command?

Reply:
The first four steps in this documentation need to be run on each bare-metal machine that requires GPU usage.

Reply:
> Where do we run the command?
The first four steps in this documentation need to be run on each bare-metal machine that requires GPU usage.

Reply:
> Where do we run the command?

The first four steps in this documentation need to be run on each bare-metal machine that requires GPU usage.

Reply:
Let me add this note to the documentation：
> "The first four steps in this document need to be run on each bare-metal machine that will use a GPU. If you've already set up each node that requires GPU usage, or you're working in a well-maintained cloud-based Kubernetes environment, you can directly start from step five."